dot/rpc/modules/api.go

```diff
@@ -19,6 +19,7 @@ type StorageAPI interface {
 	RegisterStorageChangeChannel(ch chan<- *state.KeyValue) (byte, error)
 	UnregisterStorageChangeChannel(id byte)
 	GetStateRootFromBlock(bhash *common.Hash) (*common.Hash, error)
+	Subscribe(prefixes []byte)
```

Comment:
Are we planning to subscribe to a single prefix or multiple?

Reply:
Potentially multiple.